srcpkgs/NoiseTorch/patches/dont-fetch-submods.patch

```diff
@@ -0,0 +1,11 @@
+Don't fetch submodules, since xbps-src does already.
+diff --git a/Makefile b/Makefile
+index ae252c0..6409106 100644
+--- a/Makefile
++++ b/Makefile
+@@ -28,5 +28,4 @@ release: rnnoise
+ 	rm -rf tmp/
+ 	go run scripts/signer.go -s -f bin/NoiseTorch_x64_${VERSION}.tgz
+ rnnoise:
+-	git submodule update --init --recursive
```

Comment:
Use `vsed -e '/git submodule/d' -i Makefile` in `post_extract` instead if this not going to be an upstreamed patch, the extra 10 lines are not worth it.

Reply:
wouldn't `pre_build` be better?

Reply:
I guess `post_patch` would be more correct, it should already apply `build_wrksrc` there afaik and the steps of patching and building are separated. so you you rerun just the build step during development it doesn't try to patch it every time and print a warning.